CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+#### BACKWARDS INCOMPATIBILITIES / NOTES:
```

Comment:
Agreed. I have a few more things coming before tagging this, but will bump this up to 0.3.0 at that time.